src/app/store/spinner/reducers/spinner.reducer.ts

```diff
@@ -19,5 +19,9 @@ export const spinnerState = createSelector(getSpinnerState, (state) => state.sho
 export const spinnerReducer = createReducer(
   initialSpinnerState,
   on(TechnicalRecordServiceActions.getByVIN, (state) => ({ ...state, showSpinner: true })),
```

Comment:
I think you can combine "ons" when you have multiple actions leading to the same state

on(
  TechnicalRecordServiceActions.getByVIN, 
  TestResultActions.fetchTestResultsBySystemId, 
  TestResultActions.fetchTestResults,
  (state) => ({ ...state, showSpinner: true })),

Reply:
Ah yes I was doing it in other lines as well